mlir/test/Dialect/MIOpen/translate_cflags_wrw.mlir

```diff
@@ -77,16 +77,16 @@ func @all_params(%filter : memref<?x?xf32>, %input : memref<?x?xf32>, %output :
 // CHECK: -DCK_PARAM_PROBLEM_X=3
 // CHECK: -DCK_PARAM_PROBLEM_Y=3
 // CHECK: -DCK_PARAM_TUNABLE_BLOCK_SIZE=256
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_K=2
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_M=128
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_DST_DATA_PER_WRITE_GEMM_M=1
+// CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_K=4
+// CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_M=64
+// CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_DST_DATA_PER_WRITE_GEMM_M=2
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_SRC_DATA_PER_READ_GEMM=4
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_K=8
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_N=32
+// CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_K=16
+// CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_N=16
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_DST_DATA_PER_WRITE_GEMM_N=4
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_SRC_DATA_PER_READ_GEMM=1
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_C_THREAD_COPY_DST_DATA_PER_WRITE_GEMM_N1=4
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_K_PER_BLOCK=8
+// CHECK: -DCK_PARAM_TUNABLE_GEMM_K_PER_BLOCK=16
```

Comment:
these values are changed because we introduce more tuning parameters from MIOpen, correct?

Reply:
Yes, we introduced more performance config data in the fall-back loop. I have tested it in the context of MIOpen and kernel seem to be functioning correctly there.